deps/bootstrapper/Dockerfile

```diff
@@ -23,29 +23,26 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 # Install Python and other dependencies in a single layer
 RUN pip install --no-cache-dir --upgrade pip && \
-    pip install --no-cache-dir cmake==3.22 && \
-    npm install -g --unsafe-perm ganache-cli@6.12.2
+    pip install --no-cache-dir cmake==3.22
 
-# Install specific solc version
-ENV SOLC_VERSION="0.6.12"
-ENV SOLC_PATH="/usr/local/bin/solc-$SOLC_VERSION"
+RUN npm install -g --unsafe-perm ganache@7.4.3
 
-RUN if [ ! -f "$SOLC_PATH" ]; then \
-    echo "Installing solc $SOLC_VERSION..." && \
-    apt-get update && apt-get install -y wget && \
-    wget https://github.com/ethereum/solidity/releases/download/v${SOLC_VERSION}/solc-static-linux -O "$SOLC_PATH" && \
-    chmod +x "$SOLC_PATH"; \
-    fi
+# Install specific solc version
+RUN curl https://binaries.soliditylang.org/linux-amd64/solc-linux-amd64-v0.6.12+commit.27d51765 -o /usr/local/bin/solc-0.6.12
+RUN echo 'f6cb519b01dabc61cab4c184a3db11aa591d18151e362fcae850e42cffdfb09a /usr/local/bin/solc-0.6.12' | sha256sum --check
+RUN chmod +x /usr/local/bin/solc-0.6.12
 
 # Set working directory
 WORKDIR /app
 
 # Clone repository
-RUN git clone --branch update-cairo-0.9.0 --recurse-submodules https://github.com/starknet-io/starkgate-contracts.git .
+RUN git clone --recurse-submodules https://github.com/madara-alliance/starkgate-contracts-legacy.git .
+# TODO: Remove this when PR is merged
+RUN git checkout update-0.13.3 
```

Comment:
Just a nit here, you can do this in a one-liner (should be faster too) 😃 

`git clone --branch update-0.13.3 --single-branch --recurse-submodules https://github.com/madara-alliance/starkgate-contracts-legacy.git .`

Also, I would suggest setting the branch name as an `ENV` at the beginning of the docker file just to give it more visibility, like:

`ENV BRANCH=update-0.13.3`